src/Libraries/CoreNodes/Properties/Resources.Designer.cs

```diff
@@ -158,5 +158,14 @@ internal class Resources {
                 return ResourceManager.GetString("QuadtreeConstructionNullUVSetMessage", resourceCulture);
             }
         }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Use File.FromPath -&gt; Image.ReadFromFile -&gt; Image.Pixels nodes instead..
```

Comment:
Ok. Seems it is automatically generated. That is, for a resource string, a string like "Looks up a localized string similar to {blah blah blah}." so if that resource string ends with ".", there would be two dots.